app/interfaces/api/v1/dropdown_data.rb

```diff
@@ -75,17 +75,25 @@ def unique_code
         resource :offences do
           desc 'Return all Offence-ids to be used in advocate claims (see OffenceClasses for Litigator claims).'
           params do
```

Comment:
This all looks fine to me, I guess this is the way that you define a new resource using Grape. Would you be able to go through this bit of code with me though so I understand what is going on? Thanks.

Reply:
yes, happy to talk on slack or you can review the [grape docs here](https://github.com/ruby-grape/grape) and (grape entity here)[https://github.com/ruby-grape/grape-entity]

Reply:
Cool I'll have a look at the docs and ask if I have questions.